examples/webapp/go.mod

```diff
@@ -0,0 +1,25 @@
+module github.com/honeycombio/libhoney-go/examples/webapp
+
+go 1.17
+
+require (
+	github.com/go-ozzo/ozzo-validation v3.6.0+incompatible
+	github.com/go-sql-driver/mysql v1.6.0
+	github.com/gorilla/mux v1.8.0
+	github.com/gorilla/schema v1.2.0
+	github.com/gorilla/sessions v1.2.1
+	github.com/honeycombio/libhoney-go v1.15.8
```

Comment:
Since the example is local to the codebase, maybe change the libhoney-go dependency to find in the local path? Think we can add a replace higher in the file like:

```
replace github.com/honeycombio/libhoney-go v1.15.8 => ../../..
```

Reply:
(without some further witchcraft)